src/components/chip/chip.classnames.ts

```diff
@@ -0,0 +1,67 @@
+import { ChipSize } from './types'
+import { cva } from 'class-variance-authority'
+
+export const chipBaseVariant = cva('button', {
+  variants: {
+    size: {
+      sm: ['px-600', 'py-[3px]', 'text-50', 'gap-x-400'],
+      md: ['px-800', 'py-[5px]', 'text-75', 'gap-x-400'],
+      lg: ['px-1200', 'py-[7px]', 'text-100', 'gap-x-400'],
+    },
+  },
+  compoundVariants: [{ size: 'md' }],
```

Comment:
is this even necessary to have when you already define a `defaultVariants`  with the same values?

Reply:
Not at all necessary right now. I was trying this out to check the behavior but got left out. Thanks for catching it.

Reply:
done ✅